packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2098,6 +2123,26 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   @protected
   bool get alwaysNeedsCompositing => false;
 
+  late bool _wasRepaintBoundary;
```

Comment:
The `isRepaintBoundary` property documents:

```
  /// Warning: This getter must not change value over the lifetime of this object.
```

That comment probably needs updating now that you we support having been a repaint boundary, but no longer are one?

Reply:
Done